apps/condo/domains/common/utils/serverSideBankAccountApi.js

```diff
@@ -0,0 +1,32 @@
+const get = require('lodash/get')
+const fetch = require('node-fetch')
+const conf = require('@condo/config')
+
+const bankAccountSuggestionsConfig = conf['BANK_ACCOUNT_SUGGESTIONS_CONFIG'] && JSON.parse(conf['BANK_ACCOUNT_SUGGESTIONS_CONFIG'])
+const API_URL_BANK = get(bankAccountSuggestionsConfig, 'apiUrlBank', null)
```

Comment:
More simplified names are easier to work with
```suggestion
const API_URL = get(bankAccountSuggestionsConfig, 'url', null)
```